src/listeners/guild_create.ts

```diff
@@ -0,0 +1,31 @@
+import {Events, Listener, PieceContext} from '@sapphire/framework';
+import {BediEmbed} from '../lib/BediEmbed';
+import colors from '../utils/colorUtil';
+import { Guild } from 'discord.js';
+import { surroundStringWithBackTick } from '../utils/discordUtil';
+import {getSettings } from '../database/models/SettingsModel';
+import settingsModel from "../database/models/SettingsModel";
+
+module.exports = class GuildCreateListener extends Listener {
+  constructor(context: PieceContext) {
+    super(context, {
+      once: true,
+      event: Events.GuildCreate,
+    });
+  }
+
+  public async run(guild_object: Guild) {
+    let channel = guild_object.systemChannel
```

Comment:
This is the channel that those discord made info messages get sent right?

have you checked that this works even if a server has that setting off?